packages/remix-server-runtime/server.ts

```diff
@@ -335,7 +335,7 @@ async function handleDocumentRequestRR(
       ...entryContext,
       staticHandlerContext: context,
       serverHandoffString: createServerHandoffString({
-        url: context.location.pathname + context.location.search,
+        pathname: context.location.pathname,
```

Comment:
Do we consider this handoff format public API? If so can we not re-name the property here and re-purpose the "url" name?

Reply:
Updated in [9c6cf2f](https://github.com/remix-run/remix/pull/6707/commits/9c6cf2f5a63a0542883d3484d9daffee5e8fc3f2)